builder/role_image.go

```diff
@@ -321,6 +324,13 @@ func (j roleBuildJob) Run() {
 		}
 	}
 
+	if j.builder.metricsPath != "" {
```

Comment:
Yes, what is the point to emit timings for roles that are being skipped?  We still output overall timings for all roles: https://github.com/hpcloud/fissile/blob/HCF-1099-image-metrics/app/fissile.go#L429-L432

Do you think we need the empty timing?  They just make it harder to calculate average build times, because then you have to filter them out again...